src/components/Footer.js

```diff
@@ -1,7 +1,21 @@
+import styled from 'styled-components';
+
 export default function Footer() {
 	return (
-		<footer>
-			<div></div>
-		</footer>
+		<StyledFooter>
+			<div>ᘻᗩᘜᓰᑢ ᘻᘿᗩᒪS</div>
```

Comment:
It is interesting that you didn't use a `font`. I'm not sure if this works on all browsers and devices, but it's interesting that this works on my device too. Please check how to add fonts to your repository (keyword: font-face). 😅  The semantic element should be a heading and not a div. :)

Reply:
What exactly do you mean with  heading element? <header> or <HTMLHeadingElement> ?

Reply:
Heading means element like `h1` or `h2`. [See MDN.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)